tests/src/FunctionalJavascript/MessageFormTest.php

```diff
@@ -280,4 +281,80 @@ public function testDefaultFormNotChanged(): void {
     $this->assertUrl('/');
   }
 
+  /**
+   * Asserts the sent emails are correctly formatted.
+   *
+   * @param array $captured_emails
+   *   The captured emails that were sent out: main one and auto-reply.
+   *
+   * @see self::testCorporateForm
+   */
+  protected function assertTestEmailBodies(array $captured_emails): void {
+    // First email is the outgoing one.
+    $expected = <<<EOF
+tester (not verified) (tester@example.com) sent a message using the contact
+form at http://web:8080/build/contact/oe_contact_form.
+
+
+
+      The sender's name
```

Comment:
I see the issues in drone with composer lowest, would it make sense to test for partial output with strpos like above ?

Reply:
Failure on lowest related to the difference in rendering between <=8.9.10 and >=8.9.11 version of core (and particular this change https://git.drupalcode.org/project/drupal/commit/05083ea0824377a2b41613671cf5a00b3858f696).
For making this green for both versions - it makes sense to normalize compared strings by removing more than 1 spaces, newline symbols, etc., or use the suggestion of Andras.

Reply:
Thanks @sergepavle for finding this